ERROR - 2024-10-04 15:28:16 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=11972 in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 184)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'petz24_db', 3306, '', 0)
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'petz24_db', 3306, '', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-10-04 15:28:16 --> ErrorException: Maximum execution time of 120 seconds exceeded
in SYSTEMPATH\Debug\Exceptions.php on line 202.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-10-04 15:28:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: MySQL server has gone away
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 3 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-04 15:28:16 --> ErrorException: Maximum execution time of 120 seconds exceeded
in SYSTEMPATH\Session\Handlers\FileHandler.php on line 134.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-10-04 15:29:39 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=13264 in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 184)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'petz24_db', 3306, '', 0)
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'petz24_db', 3306, '', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\projects\petzadmin\system\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
#8 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-10-04 15:29:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: MySQL server has gone away
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `pet_tbl`
WHERE `flag` = :flag:', [...], false)
 3 SYSTEMPATH\Model.php(242): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(638): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Workflow_Pet.php(19): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Pet->getPet()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Pet))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
